cloudsync/tasks.py

```diff
@@ -211,11 +212,17 @@ def update_video_statuses(self):
             refresh_status(video)
         except EncodeJob.DoesNotExist:
             # Log the exception but don't raise it so other videos can be checked.
-            log.exception("No EncodeJob object exists for video id %d", video.id)
+            log.exception("No EncodeJob object exists for video id %d",
+                          positional_args=(video.id,),
+                          video_id=video.id)
```

Comment:
Since `video_id` is a tag, no need for positional_args:
```python
log.exception("No EncodeJob object exists for video id",
                          video_id=video.id)
```

Reply:
Changed in 17e0a9e